client/server/pages/analytics.js

```diff
@@ -5,27 +5,43 @@ import analytics from '../lib/analytics';
 // Compute the number of milliseconds between each call to recordTiming
 const THROTTLE_MILLIS = 1000 / config( 'statsd_analytics_response_time_max_logs_per_second' );
 
-const logAnalyticsThrottled = throttle( function ( sectionName, duration, target ) {
-	analytics.statsd.recordTiming( sectionName, 'response-time', duration );
-	if ( target ) {
-		analytics.statsd.recordCounting( sectionName, `target.${ target }` );
```

Comment:
I removed the target stat now that we've deprecated IE11 and don't have multiple targets. I also don't think we are setting `target` on the context any more after a brief search, and given the lack of data in Grafana, this stat isn't being recorded anyways.

Reply:
We're still setting `target` [here in `getDefaultContext`](https://github.com/Automattic/wp-calypso/blob/trunk/client/server/pages/index.js#L170) but yeah, it's always `evergreen` these days.